docs/manual.tex

```diff
@@ -1,16 +1,34 @@
 \documentclass[a4paper]{report}
 \usepackage{microtype}
+\usepackage{imakeidx}
 \usepackage{hyperref}
 \usepackage{color}
 \usepackage{graphicx}
 \usepackage{eso-pic}
 \usepackage{multicol}
 \usepackage{alltt}
 
+\makeindex[intoc, columns=3]
+
 \hypersetup{pdfauthor="Ravelli/Durex", pdftitle="durexForth: Operators Manual", colorlinks=true}
```

Comment:
("ravelli" and "durex" are joke names, at some point it should be changed for something appropriate)

Reply:
The quotation marks here also make it into the PDF metadata, where they probably shouldn't be.